infrastructure/io/src/testFixtures/java/tech/pegasys/teku/cli/TempDirUtils.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2021 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.cli;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.apache.commons.io.FileUtils;
+
+public class TempDirUtils {
+
+  public static Path createTempDir() {
+    try {
+      return Files.createTempDirectory("teku_unit_test_");
```

Comment:
nit: why the final `_` ?

Reply:
That's a prefix string, so the dir name would be like `teku_unit_test_23984792347`